src/main/java/mate/academy/spring/validation/EmailValidation.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidation implements ConstraintValidator<Email,String> {
+    private static final String EMAIL_PATTERN = "^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@((\\[[0-9]{1,3}"
+            + ".\\[[0-9]{1,3}\\.\\[[0-9]{1,3}\\"
+            + ".\\[[0-9]{1,3}])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        if (email == null) {
+            return false;
+        }
+        Pattern pattern = Pattern.compile(EMAIL_PATTERN);
```

Comment:
![image](https://github.com/mate-academy/jv-spring-security/assets/20326858/28316a1a-98c9-4b8e-99e7-1489c86c61a6)

Reply:
You me Constanta private static final String EMAIL_PATTERN

Reply:
String EMAIL_PATTERN -> EMAIL_REGEX
add
private static final Pattern EMAIL_PATTERN = ...